src/input/input.js

```diff
@@ -82,7 +93,7 @@ class Input extends React.Component<InputPropsT, InternalStateT> {
           <Label
             {...getOverrideProps(LabelOverride)}
             {...sharedProps}
-            htmlFor={this.props.id}
+            htmlFor={this.getInputIdAttr()}
```

Comment:
Label was extracted, so can revert this change for an easier rebase

Reply:
Removed after the rebase.